ShestakUI/Modules/Skins/Blizzard/EncounterJournal.lua

```diff
@@ -111,6 +111,13 @@ local function LoadSkin()
 	T.SkinScrollBar(EncounterJournalEncounterFrameInstanceFrameLoreScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoOverviewScrollFrameScrollBar)
 	T.SkinScrollBar(EncounterJournalEncounterFrameInfoBossesScrollFrameScrollBar)
+	
+	EncounterJournalSuggestFrame.Suggestion1.reward:StripTextures()
+	EncounterJournalSuggestFrame.Suggestion1.reward:SetTemplate("Transparent")
```

Comment:
Every time you turn the page the icon is Suggestion1. It skins rewards from every page. There would have to be 2 rewards on the same page for there to be a need to do what you're suggesting.
I've been using this for over a week and have never seen two rewards on the same page.

But if you want to make some modifications I'm more than happy to close this, just let me know!